web-model/src/main/java/com/oasis/webmodel/response/LoginResponseModelOK.java

```diff
@@ -0,0 +1,12 @@
+package com.oasis.webmodel.response;
+
+import lombok.Data;
+
+@Data
+public class LoginResponseModelOK {
+    private String employeeId;
+
+    public LoginResponseModelOK(String employeeId){
```

Comment:
you can use lombok `@NoArgsConstructor` & `@AllArgsConstructor`
no need to create constructor manually

Reply:
Commit ID: 279a1f3